north/src/linux/inotify.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2019 - 2020 ESRLabs
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+use anyhow::{Context, Result};
+use async_std::future;
+use async_std::task;
+use nix::sys::inotify::{AddWatchFlags, InitFlags, Inotify};
+use std::fs::metadata;
```

Comment:
Isn't `rustfmt` grouping this?
Think a `cargo +nightly fmt --all` is need here.

Reply:
something went wrong with my rustfmt